ostia-operator/pkg/controller/api/api_controller_test.go

```diff
@@ -0,0 +1,87 @@
+package api
+
+import (
+	"context"
+	"testing"
+
+	ostiav1alpha1 "github.com/3scale/ostia/ostia-operator/pkg/apis/ostia/v1alpha1"
+
+	routev1 "github.com/openshift/api/route/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func testAPIControllerDeploymentCreate(t *testing.T) {
```

Comment:
I know this one is not being active, but still it is useful to get it working in the future.